CantoboardFramework/Keyboard/View/KeyCap.swift

```diff
@@ -234,7 +234,7 @@ indirect enum KeyCap: Equatable, ExpressibleByStringLiteral {
         case .space(.space): return LocalizedStrings.keyTitleSpace
         case .keyboardType(.numeric): return "123"
         case .keyboardType(.symbolic): return "#+="
-        case .keyboardType(.alphabetic): return "ABC"
+        case .keyboardType(.alphabetic): return "ABC" // schema.shortName
```

Comment:
The `shortName` should be displayed instead of `ABC` except for English.

Reply:
Could you revert this for now as this's the right approach

Reply:
Handled in #45. Will revert.